python/samples/getting_started_with_agents/chat_completion/step12_chat_completion_agent_code_interpreter.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import datetime
+import os
+
+from azure.core.credentials import AccessToken
+from azure.core.exceptions import ClientAuthenticationError
+from azure.identity import DefaultAzureCredential
+
+from semantic_kernel.agents import ChatCompletionAgent
+from semantic_kernel.connectors.ai.open_ai import AzureChatCompletion
+from semantic_kernel.contents import ChatMessageContent, FunctionCallContent, FunctionResultContent
+from semantic_kernel.core_plugins import SessionsPythonTool
+
+"""
+The following sample demonstrates how to create a chat completion agent with
+code interpreter capabilities using the Azure Container Apps session pool service.
+"""
+
+auth_token: AccessToken | None = None
+
+ACA_TOKEN_ENDPOINT: str = "https://acasessions.io/.default"
```

Comment:
This is defined in the `ACASessionsSettings` - do we need this also defined here?